verifact/forum/schema.py

```diff
@@ -55,19 +47,30 @@ class Meta:
         interfaces = (relay.Node,)
 
 
-class QuestionConnection(relay.Connection):
+class VoteConnection(relay.Connection):
     class Meta:
-        node = QuestionNode
+        node = VoteNode
 
 
-class AnswerConnection(relay.Connection):
+class AnswerNode(DjangoObjectType):
     class Meta:
-        node = AnswerNode
+        model = Answer
+        interfaces = (relay.Node,)
+        convert_choices_to_enum = False
 
+    viewer_vote = ConnectionField(VoteConnection)
```

Comment:
I think that since there will only ever be a single vote here, you should directly expose the vote instead of using relay's connections and nodes.